.woodpecker.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=81996a0479fa7c1aa7b40d96ba2bea14569d46b6
+WEB_COMMITID=0e77efc9fac45c38796ade9fbee0b6345e910e65
```

Comment:
Please use:

```suggestion
WEB_COMMITID=8c10b33af6ec3a949f95a57f197ee915d666f343
```

Reply:
Thanks. I can't figure out how I got it mixed up 🤔
<img width="1396" alt="Screenshot 2025-05-19 at 15 57 30" src="https://github.com/user-attachments/assets/e419ae21-0d92-47fa-b092-d878f4368ba3" />